LEAF_Nexus/sources/Employee.php

```diff
@@ -958,7 +954,7 @@ public function lookupAllUsersLastName(string $lastName): array
                 FROM {$this->tableName}
                 WHERE lastName LIKE :lastName
                 AND deleted = 0
-                ORDER BY {$this->sortBy} {$this->sortDir}
+                ORDER BY lastName {$this->sortDir}
```

Comment:
Will this remove funtionality that users already have?  I.e. being able to decide what to sort by?

Reply:
The sortBy variable was originally set as 'lastName' and was removed.